getting-started/forge-multi-node.md

```diff
@@ -0,0 +1,256 @@
+nodes within the Timescale Forge console, **we strongly recommend that you include 
+“AN” and “DN” in the names of each service, respectively (eg. “AN-mycluster”, 
+“DN1-mycluster”, “DN2-mycluster”, etc.)**. Again, remember that Services can only
+assume one role in a cluster (access or data node), and only one Service can act 
+as the access node.
+
+For simplicity you can start with the same hardware configuration for all the 
+Services. On Timescale Forge, Service configuration can be modified later to better 
+tune access and data node requirements.
+
+>:TIP:More advanced users might consider using larger disks on data nodes (this is 
+where the distributed hypertable data is stored) and more memory and CPU for the 
+access node.
+
+
+## Step 2: Upgrade the Services to v2.0
+
+Currently, any newly created Service in Timescale Forge still uses TimescaleDB 
+1.7.4 by default. This will be the case until we release TimescaleDB 2.0 for 
+production (sometime in late December 2020). Therefore,  to enable multi-node 
+functionality, you need to manually upgrade each TimescaleDB Service to the newest 
+ TimescaleDB 2.0 version available, currently Release Candidate 3 (RC3).
+
+To do this, we recommend using `psql` to connect to all Services created in Step 1 
+and running an extension update command.
+
+```bash
+psql -X -h {hostname} -p {port} tsdb tsdbadmin
+```
+
+There are a few things to note when running this command on Forge.
+
+ * First, note that we’ve passed the `-X ` parameter to `psql`. This prevents `psql`
+  from preloading the existing extension which would block the update command. 
+ * Second, you must connect with the `tsdbadmin` user for this update to work on 
+ your Timescale Forge Services.
+
+Once you are connected to each service, immediately run the following SQL command 
+to update TimescaleDB version to 2.0:
+
+```SQL
+ALTER EXTENSION timescaledb UPDATE TO "2.0.0-rc3"; 
+```
+
+**Note: that the above command has to be the first command to run once you’ve connected**.
+
+To verify that the update was successful, run the `\dx` command to list the version 
+of TimescaleDB that is currently active in the database. You should see “2.0.0-rc3” 
+listed under “Version” as shown below.
+
+```bash
+tsdb=> \dx
+                       List of installed extensions
+    Name     |  Version  |   Schema   |       Description                            
+-------------+-----------+------------+---------------------------------------
+ plpgsql     | 1.0       | pg_catalog | PL/pgSQL procedural language
+ timescaledb | 2.0.0-rc3 | public     | Enables scalable inserts and complex queries for time-series data
+```
+
+## Step 3: Add Data Nodes to the cluster
+
+Once you’ve created your new Services and upgraded TimescaleDB, you’ll enable 
+communication between the access node and all data nodes. The currently supported 
+method for enabling communication between nodes is through **user mapping authentication**.
```

Comment:
maybe a link pointing to PG docs: https://www.postgresql.org/docs/current/sql-createusermapping.html

Reply:
Added below on line 127 as part of the larger discussion. Can change later if a problem.